CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changelog for NeoFS Node
 
 ### Added
 - `--timeout` flag in `neofs-cli control` commands (#1917)
+- `TreeService.GetTrees` RPC; all trees synchronization on bootstrap (#1902)
```

Comment:
Why not 2 different lines? It is ok to refer to the same issue.

Reply:
ok, separated them